README.md

```diff
@@ -23,18 +24,20 @@ Pluggable prerendering stack for ahead-of-time data fetching and embedding in Pr
   Setup of prerequisites and Edge Delivery Services is guided and some steps are automated.
 
 ### Configuration Wizard
+
+  1. First install `aio` CLI globally: `npm install -g @adobe/aio-cli`.
```

Comment:
This seems to be a prereq

Reply:
I don't think we can add to package as dependency - it should be globally installed.

I will just move this instruction.